airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -64,17 +67,29 @@ type IProps = {
 } & ({ job: JobListItem } | { jobInfo: JobInfo });
 
 const JobItem: React.FC<IProps> = ({ shortInfo, ...props }) => {
-  const [isOpen, setIsOpen] = useState(false);
+  const jobMeta = isJobEntity(props) ? props.job.job : props.jobInfo;
+  const { hash } = useLocation();
+  const { jobId: linkedJobId } = parseAttemptLink(hash);
```

Comment:
I believe it could be better to extract it to separate hook and hide the details that we extract it from hash

Reply:
Good call. I think I initially didn't do it because in the other place I need it inside a state initializer only so the hook won't be benefitial, but I agree that we still get better quality by extracting it for this place.